Source/Diagnostics/ComputeDiagFunctors/BackTransformParticleFunctor.cpp

```diff
@@ -126,7 +126,6 @@ BackTransformParticleFunctor::operator () (PinnedMemoryParticleContainer& pc_dst
                 const auto& ptile_src = particles.at(index);
                 auto src_data = ptile_src.getConstParticleTileData();
                 // Flag particles that need to be copied if they cross the z-slice
-                // setting this to 1 for testing (temporarily)
                 amrex::ParallelFor(np,
```

Comment:
The call to amrex::filterParticles would do the same operation as the parallel for and it would also copy the selected particles to the dst pc.

Reply:
I agree you don't need this here because you're doing the filter and transform operations manually. If you want, you could save some lines of code by using the `filterAndTransformParticles` function in amrex: https://github.com/AMReX-Codes/amrex/blob/development/Src/Particle/AMReX_ParticleTransformation.H#L478